packages/react-components/react-rating-preview/src/components/Rating/useRating.tsx

```diff
@@ -11,8 +11,8 @@ import type { RatingProps, RatingState } from './Rating.types';
 import { RatingItem } from '../../RatingItem';
 import { StarFilled, StarRegular } from '@fluentui/react-icons';
 
-const defaultIconFilled = <StarFilled />;
-const defaultIconOutline = <StarRegular />;
+const defaultIconFilled = StarFilled;
+const defaultIconOutline = StarRegular;
```

Comment:
nit: These are not needed anymore, you can inline them back to component as icon components have stable references 😉 

P.S. The same for `RatingDisplay`